lib/manager/github-actions/index.ts

```diff
@@ -6,5 +6,5 @@ const language = LANGUAGE_DOCKER;
 export { extractPackageFile, language };
 
 export const defaultConfig = {
-  fileMatch: ['^\\.github\\/workflows\\/[^/]+\\.ya?ml$'],
+  fileMatch: ['^\\.github\\/workflows|workflow-templates\\/[^/]+\\.ya?ml$'],
```

Comment:
```suggestion
  fileMatch: ['^\\.github\\/(workflows|workflow-templates)\\/[^/]+\\.ya?ml$'],
```

Reply:
The folder needs to be in repo root of `.github` repo according to the docs